clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -69,14 +73,19 @@ const ConfigWizardWalkthrough = () => {
             </Box>
             <Box w={step === 4 ? "100%" : "40%"}>
               {step === 1 ? <OrganizationInfoForm /> : null}
-              {step === 2 ? <AddSystemForm /> : null}
+              {step === 2 ? (
+                <AddSystemForm
+                  setManualSystemsSetupChosen={setManualSystemsSetupChosen}
+                />
+              ) : null}
               {step === 3 ? <AuthenticateScanner /> : null}
               {step === 4 ? (
                 <Box pr={10}>
                   <ScanResultsForm />
                 </Box>
               ) : null}
-              {step === 5 ? (
+              {/* These steps should only apply if you're creating systems manually */}
+              {step === 5 && manualSystemSetupChosen ? (
```

Comment:
I'm a little confused why `manualSystemSetupChosen` is needed? I think step 4 goes straight to other routes, so is there a path that ends up at step 5 without going the manual route? lemme know if I'm missing something!

Reply:
You still need it because otherwise, after clicking "register selected systems" on the ScanResultsForm, it'll go to the Describe/Declare/Review page, which we are now skipping for the scanned systems flow.